README.md

```diff
@@ -247,6 +248,7 @@ updateNavigationBar = ({
 ```
 
 -   `title`: updates NavigationBar title
+-   `expandedTitle`: updates NavigationBar expandedTitle
```

Comment:
document here that empty string means no expanded title

Reply:
add some information about the minimum required app version to use this property

Reply:
Sure! Updated.